Bottom: 9183660d09d4b05a93e4d67df41cb79816a530a8
Top:    3a8d4185e06a78e9131be24a0fca2459ff407231
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 19:23:30 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
